Services/Services_bases/telegram_api_service/telegram_api.py

```diff
@@ -0,0 +1,147 @@
+#  Drakkar-Software OctoBot-Tentacles
+#  Copyright (c) Drakkar-Software, All rights reserved.
+#
+#  This library is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU Lesser General Public
+#  License as published by the Free Software Foundation; either
+#  version 3.0 of the License, or (at your option) any later version.
+#
+#  This library is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with this library.
+import logging
+
+import octobot_commons.constants as common_constants
+import octobot_commons.logging as bot_logging
+import telegram
+import telethon
+
+import octobot.constants as constants
+import octobot_services.constants as services_constants
+import octobot_services.services as services
+
+
+class TelegramApiService(services.AbstractService):
+    LOGGERS = ["TelegramApiService.client.updates", "TelegramApiService.extensions.messagepacker",
+               "TelegramApiService.network.mtprotosender", "telethon.crypto.aes", "telethon.crypto.aesctr"]
+
+    def __init__(self):
+        super().__init__()
+        self.telegram_client: telethon.TelegramClient = None
+        self.me = None
```

Comment:
I think we should name this attr in a more explicit way (to know what it is)

Reply:
It's "me" :laughing: